collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/EntityFormTest.kt

```diff
@@ -192,7 +192,7 @@ class EntityFormTest {
     }
 
     @Test
-    fun manualEntityFormDownload_withUnsupportedSpecVersion_completesSuccessfully() {
+    fun manualEntityFormDownload_withUnsupportedSpecVersion_completesSuccessfully_butThrowsAnErrorAfterOpeningIt() {
```

Comment:
I thought about that and keeping just one test, but since I didn't really favor any option, I decided to add all three. For me, the time required to run these tests has never been a significant issue, as I don’t run them in bulk locally. I always use Firebase and get results in 4-5 minutes. Additionally, we have recently removed many tests from the regression package—see the last part: https://github.com/getodk/collect/pull/6383 so I believe we have fewer tests overall. Generally, I think it’s better to have such tests, even if they take some time to run. However, as I mentioned, I was unsure about keeping or removing them, so I'm fine with keeping just one test if that’s your preference, just let me know.